crates/astria-composer/src/composer.rs

```diff
@@ -20,8 +20,14 @@ use crate::{
     Config,
 };
 
+/// Composer is a service in the Astria ecosystem responsible for submitting
```

Comment:
I think we don't need to repeat `Astria` just quite that often. :-)

```suggestion
/// Composer submits transactions to the astria shared sequencer.
```

Reply:
```
/// Composer is a service responsible for submitting transactions to the Astria
/// Shared Sequencer.
```

Is this ok? I want to make clear that the Composer is a separate service that runs on its own. Maybe redundant though, open to thoughts